tasks.py

```diff
@@ -417,10 +431,8 @@ def clean(_c, debug=False):
     if debug:
         getLogger().setLevel("DEBUG")
 
-    temp_dir = TESTS_PATH.joinpath("tmp")
-
-    for tarball in temp_dir.glob("*.tar"):
```

Comment:
No longer needed because we don't write docker images to disk locally any longer